connectors/src/connectors/intercom/temporal/activities.ts

```diff
@@ -70,9 +73,23 @@ export async function syncHelpCentersActivity({
     dataSourceName: dataSourceConfig.dataSourceName,
   };
 
-  console.log("syncHelpCentersActivity", {
-    connectorId,
-    dataSourceConfig,
-    loggerArgs,
+  const intercomClient = await getIntercomClient(connector.connectionId);
+  const helpCentersOnDb = await IntercomHelpCenter.findAll({
+    where: {
+      connectorId,
+    },
+  });
+
+  await sequelize_conn.transaction(async (transaction) => {
```

Comment:
Long lived transactions are not recommended for the DB. We likely can get away without it?

Reply:
Yes you're right, especially as the activity is retried. I added a commit to remove it!